js/util/vectortile_to_geojson.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var VectorTileFeature = require('vector-tile').VectorTileFeature;
+
+module.exports = Feature;
```

Comment:
I have a strong preference towards getting rid of `new` and using a plain function call like `convertVectorFeatureToGeoJSONFeature`